.github/workflows/_publish-mobile-github-release.yml

```
@@ -61,6 +64,10 @@ jobs:
 
           if [ "$is_latest_draft" != "true" ]; then
               echo "No draft found"
+              if ${{ inputs.dry_run}} != "true"; then
+                echo "Disabling workflow to prevent further runs"
+                gh workflow disable publish-github-release.yml
```

Since workflow is passed in this can use the input.
Since using the input directly will get flagged by the linters, you can add the env in the job as:
```
   env:
       WORKFLOW_NAME: ${{ inputs.workflow_name}
```       
```suggestion
        gh workflow disable "$WORKFLOW_NAME"
```